apps/website/contents/behavioral-interview.md

```diff
@@ -38,19 +40,129 @@ Typically, behavioral interview questions can be split into several types:
+- Staff: A story about an ambiguous project that the candidate took ownership of and was able to drive consensus from stakeholders in their org. Usually requiring two or more teams to work on.
+
+**Perseverance**
+
+Example Questions:
+
+- Tell me about a time when you needed to overcome external obstacles to complete a task or project.
+- Tell me about a time a project took longer as expected Example Responses:
+- Junior: A story about a task with many technical difficulties and how they overcame each blocker.
+- Senior: A story about a project with many technical difficulties that were blocking their team and how they overcame each blocker.
+- Staff: A story about a project with many technical difficulties that were blocking many teams and how they overcame each blocker.
+
+  **Conflict Resolution and Empathy**
+
+  Example Questions:
+
+- Tell me about a person or team who you found the most challenging to work with.
+- Tell me about a time you disagreed with a coworker.
+- Tell me about a situation where two teams couldn’t agree on a path forward.
+
+Example Responses:
+
+- Junior: A story about how they were able to work through a disagreement with a coworker on an implementation detail of a larger project.
+- Senior: A story about how they were able to work through a disagreement with a few coworkers or team leads on the direction of a larger project.
+- Staff: A story about how they were able to work through a disagreement with two or more teams on the direction of a large project.
+
+**Growth**
+
+Example Questions:
+
+- Describe a situation when you made a mistake, and what you learned from it.
+- Tell me about some constructive feedback you received from a manager or a peer
+- Tell me about a skill set that you observed in a peer or mentor that you want to develop in the next six months.
+
+Example Responses:
+
+- Junior: A story about a new technology they want to learn and the progress they have made to learn it.
+- Senior: A story about a soft skill or technical skill they want to develop and the progress they have made to learn it. Usually a skill that will have the potential to affect the entire team.
+- Staff: A story about a soft skill or technical skill they want to develop and the progress they have made to learn it. Usually a skill that will have the potential to affect two or more teams.
+
+**Communication**
+
+- Generally covered during the interview as to how clearly they are explaining the stories. There is also some overlap with Empathy and how they communicate with others.
+
+## After the interview
+
+After the interview, the interviewer will compile their feedback and give a hire/no-hire recommendation as well as the candidate’s seniority. The hire/no-hire decision is given as a spectrum, ranging from low confidence to high confidence. This is illustrated below:
+
+![The spectrum of the no-hire to hire interviewing decision overlayed on the interviewer's confidence](https://imgur.com/zRma48X)
```

Comment:
The url for this image probably needs to be replaced with your own hosted version (and not imgurs)